components/card/profile/src/_settings.scss

```diff
@@ -0,0 +1,11 @@
+$bd-card-profile-avatar-halo: 4px solid $c-accent !default;
+$bgc-card-profile: $c-gray-lightest !default;
+$c-profile-info: $c-gray !default;
+$ff-profile-rating-info-box-name: 'Open Sans', sans-serif !default;
```

Comment:
Missing context in variable name. 
Anyways, is this definition really needed? shouldn't be the font family defined at page level?  or this componente has a different font?